txnsync/bloomFilter.go

```diff
@@ -137,8 +149,10 @@ func makeBloomFilter(encodingParams requestParams, txnGroups []transactions.Sign
 			}
 		}
 
-		sizeBits, numHashes := bloom.Optimal(len(filtedTransactionsIDs), bloomFilterFalsePositiveRate)
-		result.filter = bloom.New(sizeBits, numHashes, shuffler)
+		//sizeBits, numHashes := bloom.Optimal(len(filtedTransactionsIDs), bloomFilterFalsePositiveRate)
+		//result.filter = bloom.New(sizeBits, numHashes, shuffler)
+		result.filter = bloom.NewXor(len(filtedTransactionsIDs), &s.xorBuilder)
```

Comment:
it's ok that you've used the new filter here, but you also need to add this on lines 123-124 ( or unify these somehow )